src/components/Invoices/NewInvoice/index.js

```diff
@@ -20,15 +20,88 @@ const NewInvoice = () => {
     const dueDate = new Date();
     dueDate.setDate(dueDate. getDate() + 30);
 
-    console.log(dateFormatter(todayDate));
-    console.log(dateFormatter(dueDate)); // leaving these in to check that import of dateFormatter is working. Remove once done.
-
     return (
         <>
-            <div>Created: {dateFormatter(todayDate)}</div>
-            <div>Due: {dateFormatter(dueDate)}</div>
-            <ClientDropDown />
-            <InputTable />
+            <div id="newInvoiceModal" className="modal container" tabIndex="-1">
+                <div className="modal-dialog modal-dialog-scrollable modal-dialog-centered modal-lg">
+                    <div className="modal-content">
+                        <div className="modal-header">
+                            <h5 className="modal-title fw-bold">New Invoice</h5>
+                            <button type="button" className="btn-close" data-bs-dismiss="modal"
+                                    aria-label="Close">
+                            </button>
+                        </div>
+                        <div className="modal-body overflow-auto px-3">
+                            <div className="container">
+                                <div className="row d-flex justify-content-between">
+                                    <div className="col-4">
+                                        <div>
+                                            <h6>From</h6>
+                                            <ul className="list-unstyled">
+                                                <li>Kermit the Frog</li>
+                                                <li>Kermit's Swamp</li>
+                                                <li>Beverly Hills</li>
+                                                <li>California</li>
+                                            </ul>
+                                        </div>
+                                        <div>
+                                            <h6>To</h6>
+
```

Comment:
check line spacing

Reply:
Thank you! Removed line spacing all together as I can't find a reasonable way to create lines in the middle of so many div's